src/applications/mhv-medical-records/containers/DownloadReportPage.jsx

```diff
@@ -26,10 +55,34 @@ const DownloadReportPage = () => {
         text="Select records and download"
       />
       <h2>Other reports you can download</h2>
+      {CCDRetryTimestamp ? (
+        <va-alert
+          close-btn-aria-label="Close notification"
+          status="error"
+          visible
+        >
+          <h2 slot="headline">
+            We can’t download your Continuity of Care Document right now
+          </h2>
+          <p>
+            We’re sorry. There’s a problem with our system.{' '}
+            <strong>Try again after 24 hours ({CCDRetryTimestamp} ET).</strong>
```

Comment:
We shouldn't be setting this date to ET. It should appear in the user's time zone, unless UCD specified otherwise.